react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -142,13 +161,36 @@ class Filmstrip extends Component <Props> {
         // will get updated without replacing the DOM. If the known DOM gets
         // modified, then the views will get blown away.
 
+        const remoteVideosStyle = { };
+        const filmstripRemoteVideosContainerStyle = {};
+        let remoteVideoContainerClassName = 'remote-videos-container';
+
+        switch (this.props._currentLayout) {
+        case LAYOUTS.VERTICAL_FILMSTRIP_VIEW:
+            // Adding 8px - margin & borders; 7px - scrollbar
```

Comment:
This comment isn't clear to me. Where are 8px and 7px in the below line?

Reply:
Those are from the thumbnail - margin-left/right:2px;border:2px. Otherwise on the next line we add 15 = 8 + 7 because of this.

Reply:
I see. I think I get it now. Really you should just use complete sentences. Something like "adding 8px for the 2px margins and 2px borders on the left and right. Also adding 7px for the scrollbar."

Reply:
Fixed